be/Cf.WebApi/Endpoints/ServiceDocumentEndpoints.cs

```diff
@@ -17,44 +19,25 @@ public static void MapServiceDocumentRoutes(this IEndpointRouteBuilder builder)
             .WithTags(Tag)
         .HasApiVersion(1);
 
-        group.MapPost("UploadPdf", UploadPdfAsync);
         group.MapPost("UploadpdfBytes", UploadPdfBytesAsync);
     }
 
     [Authorize(Roles = "Service")]
-    private static async Task UploadPdfAsync([FromServices] IServiceDocumentService pdfService, IHttpContextAccessor httpContextAccessor, [FromForm] IFormFile file)
+    private static async Task UploadPdfBytesAsync([FromServices] IServiceDocumentService pdfService, IHttpContextAccessor httpContextAccessor, [FromBody] string base64Content)
     {
-        if (file == null || file.Length == 0)
+        if (string.IsNullOrEmpty(base64Content))
         {
-            throw new ApplicationException();
+            throw new BadRequestException(DomainErrors.Service.NotFound);
         }
 
-        // Add your logic to save the PDF file using the pdfService
-        // For example, you can save it to a specific folder or a database
-        var pdfBytes = await ReadFileBytesAsync(file);
-
-        // Your PDF service method to save the PDF
-        await pdfService.SavePdfAsync(pdfBytes, GetServiceId(httpContextAccessor));
-    }
-
-    [Authorize(Roles = "Service")]
-    private static async Task UploadPdfBytesAsync([FromServices] IServiceDocumentService pdfService, IHttpContextAccessor httpContextAccessor, [FromBody] byte[] pdfBytes)
-    {
-        if (pdfBytes == null || pdfBytes.Length == 0)
+        try
         {
-            throw new ApplicationException();
+            byte[] pdfBytes = Convert.FromBase64String(base64Content);
+            await pdfService.SavePdfAsync(pdfBytes, GetServiceId(httpContextAccessor));
```

Comment:
We need to adjust this as in this place I think we still haven't created service, so one approach would be after successful registration service first of all needs to provide additional data to create `Service` in the db and only after this, it should be able to upload the document for the verification.

Reply:
I agree, will try to implement it in this pr then